### PR TITLE
Parse connection postgres uri

### PR DESCRIPTION
**Pull request by isaactovsky**
Hi, first, thanks for this library.
Some deployment platforms offer to include postgres service (heroku one example) and then expose the postgres connection uri via environment variable.
This is a util function `Uri -> Maybe Pool` that i found personally useful in my project.

### CODE REVIEW

File: src/Database/PostgreSQL.purs [hunk 1]
```diff
@@ -69,6 +76,27 @@ defaultPoolConfiguration database =
...
+  password <- tillChar (char '@')
+  host <- tillChar (char ':')
+  port <- tillChar (char '/')
+  database <- many anyChar
+  pure { database: toStr database
+       , host: Just $ toStr host
+       , idleTimeoutMillis: Nothing
+       , max: Nothing
+       , password: Just $ toStr password
+       , port: fromString $ toStr port
```
Comment (paluh) [inline src/Database/PostgreSQL.purs:95]: Could you please move closing `}` to the new line?

File: test/Main.purs [hunk 2]
```diff
@@ -375,3 +381,18 @@ noSuchDatabaseConfig =
 cannotConnectConfig :: PoolConfiguration
 cannotConnectConfig =
   config { host = Just "127.0.0.1", port = Just 45287 }
+
+validUriToPoolConfigs :: { uri :: PgConnectionUri, poolConfig :: PoolConfiguration }
```
Comment (paluh) [inline test/Main.purs:385]: Could you please format both records using this scheme:
``` purescript
{ field1: value1
, field2: value2
...
}
```

File: README.md [hunk 3]
```diff
@@ -166,8 +166,6 @@ To run suite please:
 
 Till we are hosted on the github platform let's just use github releasing model for tagging new versions and `github-release-notes` to generate CHANGELOG.md from it:
 
-```bash
-$ # This only requires repo access
-$ export GREN_GITHUB_TOKEN=...
-$ github-release-notes changelog --override
-```
```
Comment (paluh) [inline README.md:173]: Was anything wrong with this single quote block what required this change?

**Comment by paluh** [comment]
Sorry for the late response - I'm not sure why I have not received notification about your PR.

This is great addition. Thanks for implementing this parser and tests! I've found some minor formatting issues - sorry for being so pedantic. If you don't want to fix them I can merge and fix them myself :-)

P.S.
If you are using PostgreSQL with PureScript you can be also interested in purescript-selda - please check it out!

File: README.md (hunk 3 above)
Comment (isaactovsky) [reply README.md:173]: Hi, yes, for me it generated invalid syntax for Readme module with that block. I already don't remember exact error

**Comment by isaactovsky** [comment]
Hi, on the contrary, I appreciate your feedback and guidelines about formatting records. I'm getting used to writing purescript :)

**Comment by isaactovsky** [comment]
Hi, @paluh  Any updates about when this will be merged?

**Comment by paluh** [comment]
Holy cow! I'm really sorry @isaactovsky for this delay. I had not clicked "submit review" button and it was hanging so long. I added another comment related to formatting. I hope you don't mind.
Of course if you don't have time to fix this minor issue I'm going to merge it and provide a fix myself.

**Comment by paluh** [comment]
Thanks a lot!

**Comment by paluh** [comment]
It seems that github doesn't like block splitting - this breaks [release](https://github.com/rightfold/purescript-postgresql-client#releasing) section. I'm going to test this myself before the next release.

**Comment by paluh** [comment]
@isaactovsky I've checked now node-postgresql documentation and it seems that it supports postgresql uri on its own level. I wonder if we want to parse it on our side an maintain this parsing (for example now we have an issue with `sslmode` parsing).

Maybe we should drop parser and go with a sum type like:

`data PoolConfiguration = PgConfigurationlUri String | PgConfig { ... }`

Would this kind of solution work for you?

**Pull request merged**